Packs/AbuseDB/Scripts/AbuseIPDBPopulateIndicators/AbuseIPDBPopulateIndicators_test.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from CommonServerPython import DemistoException
+
+
+def test_get_contents():
+    from AbuseIPDBPopulateIndicators import get_contents
+
+    assert get_contents({}) is None
```

Comment:
Change as we talked - use `mocker.patch('execute_command', return_value=[{'Contents': {}}])` with the return value you want to check (once with nothing, second time with a real response).
I would also send `args` with real values - you can use the default values that are in the yml file.